src/main/java/seedu/address/logic/commands/DisplayDriverTasksCommand.java

```diff
@@ -0,0 +1,36 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.person.DriverIsTheSamePredicate;
+
+/**
+ * Display all the tasks assigned to the driver given by the input index.
+ * This command is a follow up search after the FindDriverCommand.
+ */
+public class DisplayDriverTasksCommand extends Command {
+
+    public static final String COMMAND_WORD = "driver";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all the tasks assigned to the driver given by "
+            + "the specified index and displays them as a list with index numbers.\n"
+            + "Parameters: [INDEX NUMBER]\n"
+            + "Example: " + COMMAND_WORD + " 1";
```

Comment:
we aren't using index anymore, its always been using the unique id of the driver / customer / task.

Reply:
The index here refers to the index displayed in the list when it is filtered